src/iohandler/io_handler_buffer_helper.cc

```diff
@@ -43,21 +43,9 @@ IOHandlerBufferHelper::IOHandlerBufferHelper(std::shared_ptr<Config> config, siz
     if (initialFillSize > bufSize)
         throw_std_runtime_error("initialFillSize must be lesser than or equal to the size of the buffer");
 
-    this->bufSize = bufSize;
-    this->initialFillSize = initialFillSize;
+    bufSize = bufSize;
+    initialFillSize = initialFillSize;
```

Comment:
I'm not sure whether you can drop the `this->`. But you can move the init up to the init list and keep the checks. Makes it all more compact.

Reply:
Fixed.